operations/orderitems.md

```diff
@@ -344,21 +341,21 @@ Returns all order items. At least one of the `OrderItemIds`, `ServiceOrderIds`,
 | `AgeCategoryId` | string | optional | Unique identifier of the [Age Category](agecategories.md#age-category). |
 
 #### Order item options
+Options of the order item.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `CanceledWithReservation` | bool | required | Order item was canceled with reservation cancelation. |
-
+| `CanceledWithReservation` | boolean | required | Order item was canceled with reservation cancellation. |
 
-#### Accounting item state
+#### Order item accounting state
```

Comment:
I think this is gonna create problems in where it is referenced and not changed 🤔. Like here https://github.com/MewsSystems/gitbook-connector-api/blob/e1572153ee048f07ba772d797c30de2202288f60/operations/reservations.md?plain=1#L337

Reply:
Moved it to overrides - it will point to accountingitems page, as a shared type.